<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newObjectMaster_2</class>
 <widget class="QWidget" name="newObjectMaster_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Цвет линии графика</string>
   </property>
   <widget class="lineButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>График работы</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>1 Смена</string>
    </property>
    <widget class="QTimeEdit" name="startTimeEdit_1">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>22</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>5</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Начало</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>178</x>
       <y>5</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Окончание</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="stopTimeEdit_1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>22</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>2 Смена</string>
    </property>
    <widget class="QCheckBox" name="shiftCheckBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTimeEdit" name="startTimeEdit_2">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>22</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>5</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Начало</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>178</x>
       <y>5</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Окончание</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="stopTimeEdit_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>22</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>3 Смена</string>
    </property>
    <widget class="QCheckBox" name="shiftCheckBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTimeEdit" name="startTimeEdit_3">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>22</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>5</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Начало</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>178</x>
       <y>5</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Окончание</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="stopTimeEdit_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>22</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lineButton</class>
   <extends>QPushButton</extends>
   <header location="global">serviceClasses/linebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
